cluster/gce/util.sh

```diff
@@ -660,7 +660,17 @@ function create-network() {
       --project "${PROJECT}" \
       --network "${NETWORK}" \
       --source-ranges "10.0.0.0/8" \
-      --allow "tcp:1-65535,udp:1-65535,icmp" &
+      --allow "tcp:1-2379,tcp:2382-65535,udp:1-65535,icmp" &
```

Comment:
I agree, but I'm not sure how to implement it. Firewall rules on GCE allow only to allow certain tags. So, we can allow nodes as they have NODE_TAG, but we cannot allow everything but MASTER_TAG.